### PR TITLE
OCPQE-27808: update filter for baremetal-upi-ovn-ipv4-disconnected-arm

### PR DESCRIPTION
**Pull request by Xia-Zhao-rh**
update filter due to https://github.com/openshift/release/pull/59178 

@kuiwang02 
PTAL, thanks
cc @sgoveas

### CODE REVIEW

**Comment by Xia-Zhao-rh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-upi-ovn-ipv4-disconnected-arm-f14

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly.yaml [hunk 1]
```diff
@@ -1364,6 +1364,7 @@ tests:
     env:
       AUX_HOST: openshift-qe-metal-ci.arm.eng.rdu2.redhat.com
       E2E_RUN_TAGS: '@disconnected'
+      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;Serial;Disruptive;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
```
Comment (kuiwang02) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly.yaml:1367]: @Xia-Zhao-rh it is discconnected, not destructive

so
```yaml
TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
```

same comment for others

**Pull request merged**